userguide/services.rst

```diff
@@ -254,6 +249,18 @@ Asigra management application.
    the application to open *DSOP.jnlp* with, select Java Web Start
    Launcher (:command:`javaws`).
 
+While the service is running, the :guilabel:`Open DS-Operator Web Interface`
+button appears in :menuselection:`Services --> Asigra DS-System`. Click
```

Comment:
```suggestion
button appears in
:menuselection:`Services --> Asigra DS-System`.
Click :guilabel:`Open DS-Operator Web Interface` to download and launch the
```

Reply:
We're supposed to put :menuselection:`` items on their own line now (confirm with Warren)